biojava-core/src/test/java/org/biojava/nbio/core/util/SequenceToolsTest.java

```diff
@@ -0,0 +1,74 @@
+package org.biojava.nbio.core.util;
+
+import static org.junit.jupiter.api.Assertions.assertAll;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import java.util.Random;
+
+import org.junit.jupiter.api.Disabled;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Nested;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
+
+class SequenceToolsTest {
+
+	String randomDNA(int n) {
+		String[] nucs = new String[] { "A", "T", "C", "G" };
+		Random r = new Random();
+		StringBuilder sb = new StringBuilder();
+		for (int i = 0; i < n; i++) {
+			sb.append(nucs[r.nextInt(4)]);
+		}
+		return sb.toString();
+	}
+
+	@Nested
```

Comment:
I thought using nested tests  for these utility classes of  diverse static methods would help group related tests together better.